gcp_variant_transforms/libs/bigquery_schema_descriptor.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A dict based description for BigQuery's schema."""
+
+from __future__ import absolute_import
+
+from collections import namedtuple
+from apache_beam.io.gcp.internal.clients import bigquery  # pylint: disable=unused-import
+
+__all__ = ['SchemaDescriptor']
+
+# Stores data about a simple field (not a record) in BigQuery Schema.
+FieldDescriptor = namedtuple('FieldDescriptor', ['type', 'mode'])
+
+
+class SchemaDescriptor(object):
```

Comment:
In the PR description, you talk about providing "serialization" for BigQuery schema, did you mean string serialization? In any case, this class in its current form does not do serialization, does it?

Reply:
What I meant is that since this class is just a dict (or two), it can be serialized in beam. I think someone said TableSchema cannot be pickled? It's a minor fact, and I would delete it if it creates confusion. Should I?